common4j/src/main/com/microsoft/identity/common/java/opentelemetry/SpanName.java

```diff
@@ -67,5 +67,10 @@ public enum SpanName {
     DeleteTransferToken,
     RestoreMsaAccounts,
     OnUpgradeReceiver,
-    UpgradeDeviceRegistration
+    UpgradeDeviceRegistration,
+
+    /**
+     * Span covering interrupt handling using PRT protocol.
+     */
+    PrtHandleInterrupt
```

Comment:
We already had a boolean to indicate is_interrupt_flow. Is this new span a replacement of that?

Reply:
wanted to capture interrupt execution separate but I guess it's not adding reasonable value and would incur cost.